client/src/app/employees/controllers.js

```diff
@@ -43,49 +44,54 @@ angular.module('app.employees.controllers', [])
           })
           .catch(function (x) {
             employee.deleted = true;
-            console.log('error : ' + x);
+            console.log('error : ' + JSON.stringify(x));
           });
       };
 
-      $scope.cancel = function cancel () {
+      vm.cancel = function cancel () {
         $state.go('app.employees', {}, {reload: true});
       };
 
-      $scope.requestEmployees(1);
+      vm.requestEmployees(1);
     }
   )
 
   .controller('EmployeeDetailCtrl', 
-    function ($scope, $state, $stateParams, employee) {
-      $scope.saveText = $state.current.data.saveText;
-      $scope.employee = employee;
+    function ($state, $stateParams, employee) {
+      var vm = this;
 
-      $scope.save = function save () {
-        $scope.employee.$update()
+      vm.saveText = $state.current.data.saveText;
+      vm.employee = employee;
+
+      vm.save = function save () {
+        vm.employee.$update()
           .then(function (updated) {
-            $scope.timesheet = updated;
+            vm.timesheet = updated;
+            $state.go('app.employees', {}, {reload: true});
             console.log('success!');
           })
           .catch(function (x) {
-            console.log('error : ' + x);
+            console.log('error : ' + JSON.stringify(x));
```

Comment:
Not to be nitpicky, but it would be nice to have a standard way of printing this out. I went with `console.log('error: ' + JSON.stringify(x));`